content-services/content-sources-frontend
# THEEDGE-3682: Create a new modal to confirm custom repository delete action

## Summary
This PR will create a new modal to confirm the delete action for a custom repository according this mockup (https://www.sketch.com/s/ac25729e-df9b-4e74-a09c-badb62787d24/a/rbg7JMD#Inspect).

A UI example of this change could be check on this video


https://github.com/content-services/content-sources-frontend/assets/5436271/001f314e-8337-4394-a40f-04ecb92844e3

The code was made following the pattern used on edit custom repository. 
 

## Testing steps

1. Run the branch 
2. Access the path `/preview/insights/content` 
3. Click the kebab menu 
4. select the option `Delete` 
5. The new modal will show up and bring more details about the repository 
6. if you click on cancel, it will not affect the repository
7. if you confirm the remove option, it will close the modal and remove the repository
